src/views/mainGridView.jsx

```diff
@@ -19,8 +19,9 @@ export const MainGridView = ({
     peer => peer.roleName?.toLowerCase() !== ROLES.TEACHER
   );
   const hideSidePane =
-    (teacherPeers.length > 1 && studentPeers.length === 0) ||
-    (teacherPeers.length === 0 && studentPeers.length > 1);
+    peers.every(peer => peer.roleName === peers[0].roleName) &&
+    peers.filter(peer => !peer.isLocal && (peer.audioTrack || peer.videoTrack))
```

Comment:
peers.every here as well would be more efficient/better

Reply:
Done. Changed to `.some`